osu.Game/OsuGameBase.cs

```diff
@@ -96,7 +96,7 @@ public partial class OsuGameBase : Framework.Game, ICanAcceptFiles, IBeatSyncPro
         /// <summary>
         /// The maximum volume at which audio tracks should playback. This can be set lower than 1 to create some head-room for sound effects.
         /// </summary>
-        private const double global_track_volume_adjust = 0.8;
+        private const double global_track_volume_adjust = 1;
```

Comment:
Now that we've implemented normalization, this constant should probably be dropped.